Just now added corresponding registry key  and  download link for tem… (#4277)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/windows-defender-security-center/wdsc-hide-notifications.md

>
>You must have Windows 10, version 1709 or later. The ADMX/ADML template files for earlier versions of Windows do not include these Group Policy settings. 

> 0. Download the latest Administrative templates for windows 10 v1809 from below Microsoft official site

Reviewer: Although `0` is a valid list number in programming and mathematics, it is not a practical number in literature. If you want to add something to the beginning of the list, make sure that it starts with 1 and that the list further down follows order.

Author: @illfated . Thanks sir . I noted Down on the paper. I take care of these words.

> 0. Download the latest Administrative templates for windows 10 v1809 from below Microsoft official site

Reviewer 2: ```suggestion
1. Download the latest [Administrative Templates (.admx) for Windows 10, v1809](https://www.microsoft.com/download/details.aspx?id=57576).
```

>      **https://www.microsoft.com/en-us/download/details.aspx?id=57576**

Reviewer 2: ```suggestion
```

> 

Reviewer 2: ```suggestion
```

> 1.  On your Group Policy management machine, open the [Group Policy Management Console](https://technet.microsoft.com/library/cc731212.aspx), right-click the Group Policy Object you want to configure and click **Edit**.

Reviewer 2: ```suggestion
2.  On your Group Policy management machine, open the [Group Policy Management Console](https://technet.microsoft.com/library/cc731212.aspx), right-click the Group Policy Object you want to configure and click **Edit**.
```

> 1.  On your Group Policy management machine, open the [Group Policy Management Console](https://technet.microsoft.com/library/cc731212.aspx), right-click the Group Policy Object you want to configure and click **Edit**.

Reviewer 2: ```suggestion
2.  On your Group Policy management machine, open the [Group Policy Management Console](https://technet.microsoft.com/library/cc731212.aspx), right-click the Group Policy Object you want to configure and click **Edit**.
```

Reviewer 2: The link in this step is outdated - maybe there is a better one?

This can only be done in Group Policy.

>[!IMPORTANT]

> 

Reviewer 2: ```suggestion
```

> 

Reviewer 2: ```suggestion
```

6.  Open the **Hide all notifications** setting and set it to **Enabled**. Click **OK**.

> 7.  Corresponding registry key  for **Hide all notifications**

Reviewer 2: ```suggestion
7.  Use the following registry key and DWORD value to **Hide all notifications**.
```

   
    **[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows Defender Security Center\Notifications]**
    **"DisableNotifications"=dword:00000001**
    

> 8.   Corresponding registry key  for **Hide not-critical notifications**

Reviewer 2: ```suggestion
8.  Use the following registry key and DWORD value to **Hide not-critical notifications**
```
